mandatory/3-function-output.js

```diff
@@ -1,16 +1,20 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// Whenever this function is called, it generates a random number and multiplies it by 10.
 function getRandomNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+//This function takes two parameters which are strings and concatenates them into one string.
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
   // Write the body of this function to concatenate three words together.
   // Look at the test case below to understand what this function is expected to return.
+  // We could not use concat() because it only accepts strings. It would not meet all conditions of the test.
+  return firstWord + " " + secondWord + " " + thirdWord;
```

Comment:
How would you explain what happens in this function in order for the last test case to work with a number? 

Google "javascript coercion" if you'd like to find out more! We'll see more examples of this throughout the course.

Reply:
I would have to look it up. But my guess is that it has to do something with adding numbers and strings together. 13 is a number in the last test and the rest are strings. Probably the typeof would be different and they would be added together but not computed? I will look into javascript coercion. Thanks Paul.